pkg/ingestor/parser/scorecard/parser_scorecard.go

```diff
@@ -93,11 +93,11 @@ func getPredicates(s *sc.JSONScorecardResultV2) (*model.ScorecardInputSpec, *mod
 	idx := strings.LastIndex(s.Repo.Name, "/")
 	if idx < 0 {
 		name = s.Repo.Name
+	} else {
+		ns = s.Repo.Name[:idx]
+		name = s.Repo.Name[idx+1:]
 	}
 
-	ns = s.Repo.Name[:idx]
-	name = s.Repo.Name[idx+1:]
-
```

Comment:
This change looks like it fixed a bug with packages with no namepace (no / in the name)